content/03.results.md

```diff
@@ -181,31 +183,19 @@ Points are colored by `cancer_group` of the samples they represent. B, Heatmap o
+Indeed, we observed higher scores in LFS tumors (N = 8) for which we detected high-confidence _TP53_ somatic alterations. 
+Although we were unable to detect canonical somatic _TP53_ mutations in two LFS patient tumors with low _TP53_ scores, we confirmed the LFS diagnosis from pathology reports and found each to have a germline variant in _TP53_.
+The tumor purity of these samples was low (16% and 37%), suggesting the classifier requires a certain level of tumor purity to achieve good performance, as we expect _TP53_ to be intact in normal cells.
+Tumors with the highest median _TP53_ scores were those known to harbor somatic _TP53_ alterations: choroid plexus tumors, embryonal tumors, HGATs, and pineal tumors (**Figure {@fig:Fig5}E**), while melanocytic tumors, meningiomas, and tumors of cranial and paraspinal nerves had the lowest scores.
+	
 We next used gene expression data to predict telomerase activity using EXpression-based Telomerase ENzymatic activity Detection (EXTEND) [@doi:10.1038/s41467-020-20474-9] as a surrogate measure of malignant potential [@doi:10.1038/s41467-020-20474-9; 10.1093/carcin/bgp268].
-EXTEND scores significantly correlated with _TERT_ and _TERC_ expression (**Figure {@fig:FigS5}C-D**).  
+EXTEND scores significantly correlated with _TERT_ (R = 0.55) and _TERC_ (R = 0.58) expression (**Figure {@fig:FigS5}G-H**).  
 We found aggressive tumors such as CNS lymphoma, ETMR, ATRT, DMG, and HGG had high EXTEND scores (**Figure {@fig:Fig5}G**), while benign lesions such as teratomas, dysplasias, and hemangioblastomas had the lowest scores (**Table S3**).
 
-<!-- TODO: Add link to S3 once generated. -->
-
-![**Figure 5. _TP53_ and telomerase activity** A, Receiver Operating Characteristic for _TP53_ classifier run on FPKM of stranded RNA-Seq samples.
-B, Violin and box plots of _TP53_ scores plotted by _TP53_ alteration type.
-C, Violin and box plots of _TP53_ RNA expression plotted by _TP53_ activation status.
-D, Violin and box plots of _TP53_ scores in Li Fraumeni patient tumors.
-E, _TP53_ scores significantly correlate with CNV breakpoint density across PBTA tumors (Pearson R = 0.25, p = 7.5e-12).
-F, Box plots of _TP53_ scores grouped by `broad_histology`.
-G, Box plots of EXTEND scores grouped by `broad_histology`. 
-H, _TP53_ scores significantly correlate with EXTEND scores across PBTA tumors (Pearson R = 0.31, p <2.2e-16).
-I, Histology-specific Pearson correlations between _TP53_ and EXTEND scores.
-J, Kaplan-Meier](images/main/Figure5.png){#fig:Fig5}
+![**Figure 5. _TP53_ and telomerase activity** A, Receiver Operating Characteristic for _TP53_ classifier run on FPKM of stranded RNA-Seq samples. B, Violin and box plots of _TP53_ scores plotted by _TP53_ alteration type. C, Violin and box plots of _TP53_ RNA expression plotted by _TP53_ activation status. D, Box plots of _TP53_ scores grouped by `broad_histology`. E, Box plots of EXTEND scores grouped by `broad_histology`.](images/main/Figure5.png){#fig:Fig5}
```

Comment:
We may have to come back to this later when fig 5 is finalized. Maybe add a comment reminder in here just in case?

Reply:
sure